xAPI.md

```diff
@@ -2639,34 +2658,48 @@ such a mechanism.
 * The LRS SHOULD at a minimum supply OAuth with "HMAC-SHA1" and "RSA-SHA1" signatures.
 
 ###### Application registered + known user Process and Requirements
+**Process:** The standard workflow for OAUth 1.0 is used. 
 
-* Use endpoints in section [6.4.2 OAuth Authorization Scope](#oauthscope) to complete the standard OAuth workflow 
+**Requirements:**
+* The LRS MUST support the endpoints in section [6.4.2 OAuth Authorization Scope](#oauthscope) to complete the standard OAuth workflow 
 (details not in this specification).
 * If this form of authentication is used to record Statements and no authority is specified, the LRS SHOULD 
 record the authority as a group consisting of an Agent representing the registered application, and an Agent 
 representing the known user.
 
 ###### Application registered + user unknown Process and Requirements
+**Process:** 
+The LRS honors requests that are signed using OAuth with the registered application's credentials and with an empty token and token secret.
 
-* The LRS honors requests that are signed using OAuth with the registered application's credentials and with an empty token and token secret.
-* If this form of authentication is used  to record Statements and no authority is specified, the LRS SHOULD 
+**Requirements:**
+* If this form of authentication is used to record Statements, the LRS SHOULD 
 record the authority as the Agent representing the registered application.
 
 ###### Application not registered + known user Process and Requirements
+**Process:**
+The AP uses a consumer secret consisting of an empty string to call the 
+Temporary Credential Request endpoint specifying the "consumer_name" and other usual parameters.
+The "consumer_name" contains a string representing the application requesting access. 
+
+The AP then sends the user's browser to the Resource Owner Authorization using the 
+temporary credentials obtained from the LRS. The Resource Owner Authorization presents a 
+page displaying the "consumer_name" plus a warning that the identity of the application requesting 
+authorization cannot be verified.
 
-* The AP MUST use a consumer secret consisting of an empty string.
-* Call "Temporary Credential" request.
-* Specify "consumer_name" and other usual parameters; User will then see "consumer_name" plus a warning 
-that the identity of the application requesting authorization cannot be verified.
-* The LRS MUST record an  authority that includes both that application and the authenticating user, as a group, 
-since OAuth specifies an application.
+Otherwise the process follows the standard OAuth workflow. 
+
+**Requirements:**
+* If this form of authentication is used to record Statements, the LRS SHOULD record an authority 
```

Comment:
Added ```If this form of authentication is used  to record Statements``` for consistency with the above. 

TODO: Make the SHOULD back to a MUST. I was thinking that the LRS might trust the application to specify the authority but since the scenario is specifically an unknown application, that's obviously not the case.